data/json/monster_special_attacks/spells.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "id": "pain_damage",
+    "type": "SPELL",
+    "name": "Pain",
+    "description": "Increases pain",
+    "valid_targets": [ "hostile" ],
+    "//": "Listed as a recover energy effect with a negative modifier that decreases with each level of the spell, which makes it cause damage instead.",
+    "min_damage": -25,
+    "max_damage": -225,
+    "damage_increment": -50,
+    "effect": "recover_energy",
+    "effect_str": "PAIN"
+  },
+  {
+    "id": "stamina_damage",
+    "type": "SPELL",
+    "name": "Tired",
+    "description": "decreases stamina",
+    "valid_targets": [ "hostile" ],
+    "//": "Listed as a recover energy effect with a negative modifier that decreases with each level of the spell, which makes it cause damage instead.",
+    "min_damage": -2000,
+    "max_damage": -20000,
```

Comment:
stamina itself can be up to 10,000. is it intended to put the player this far into the negatives (potentially)?